lib/hummingbird_survey/models/survey_page_form.rb

```diff
@@ -153,14 +153,23 @@ def build_final_page_data
       result
     end
 
+    def answer_value(value, item)
+      case item.survey_itemable.question_type
+      when SurveyQuestionType::Encrypted.new
```

Comment:
@cgratigny : This is a totally valid way of doing it, but up until this PR, we've been using the format `SurveyQuestionType[:encrypted]`. Not sure how important that particular consistency is.